packages/ckeditor5-html-support/src/converters.js

```diff
@@ -106,7 +112,11 @@ export function viewToAttributeInlineConverter( { view: viewName, model: attribu
 					conversionApi.writer.setAttribute( attributeKey, attributesToAdd, node );
 				}
 			}
-		}, { priority: 'low' } );
+		}, {
+			// With the low - 10 priority to execute after the element-to-attribute converters and their fallback converters
+			// for consuming the element itself so after consuming <a href="..."> both an attribute and the element should be consumed.
```

Comment:
I'm having trouble parsing this. It's a _very_ long sentence 😅 

It says `both an attribute and the element should be consumed` → so only a single attribute, not all?